exercise13_1.py

```diff
@@ -8,24 +8,31 @@
 Python for Everybody: Exploring Data Using Python 3
 by Charles R. Severance
 
-Solution by Jamison Lahman, June 5, 2017
+Solution by Daniel Lee, January 5, 2019
 """
 
 import urllib.request
 import urllib.parse
 import urllib.error
 import json
 
-# Note that Google is increasingly requiring keys
-# for this API
-serviceurl = 'http://maps.googleapis.com/maps/api/geocode/json?'
+api_key = False
+# If you have a Google Places API key, etner it here
+#api_key = 'xxx'
+
+if api_key is False:
```

Comment:
```suggestion
if not api_key:
```
Otherwise, this seems like a boolean.